share/completions/mkvextract.fish

```diff
@@ -31,16 +35,18 @@ function __fish_mkvextract_find_matroska_in_args
 end
 
 function __fish_mkvextract_print_attachments
-    set -l matroska (__fish_mkvextract_find_matroska_in_args)
-    if test -n '$matroska'
-        mkvmerge -i $matroska | string match 'Attachment ID*' | string replace -r '.*?(\d+).*? type \'(.*?)\'.*?file name \'(.*?)\'' '$1:\t$3 ($2)'
+    if set -l matroska (__fish_mkvextract_find_matroska_in_args)
+        if set -l info (mkvmerge -i $matroska)
+            string match 'Attachment ID*' $info | string replace -r '.*?(\d+).*? type \'(.*?)\'.*?file name \'(.*?)\'' '$1:\t$3 ($2)'
```

Comment:
Ideally, you'd guard the "$info" from being interpreted as options: `string match 'Attachment ID*' -- $info`. Otherwise if a line is e.g. `-r`, string will go into regex mode.